CONTRIBUTING.md

```diff
@@ -1 +1,2 @@
-See ["Contributing to Ruby"](https://docs.ruby-lang.org/en/master/contributing_md.html), which includes setup and build instructions.
+See ["Contributing to Ruby"](contributing/contributing.md),
```

Comment:
This link is broken on GitHub's preview, so we want to use an absolute URL.

Reply:
Can I see the broken link?

Reply:
Click the link I shared above. Then click the `"Contributing to Ruby"` link. It's a 404.

Reply:
> Click the link I shared above. Then click the `"Contributing to Ruby"` link. It's a 404.

I'm happy to make the change you suggest, but would like to understand what's going on here.  Why is this not a problem elsewhere?

Reply:
This is because this file exists in both places. `CONTRIBUTING.md` is a special file in GitHub but it's also rendered via RDoc in docs.ruby-lang.org.

Reply:
Fixed (now links to 3.4).

Reply:
Can you link to the master branch docs instead of a release version? That would be the most up-to-date.

Reply:
Done (committed here).

Reply:
The new link is dead.

Reply:
Sorry.  It was careless of me to not check.  I must confess, though, that I don't know what the link target should be;  any help?

Reply:
It's https://docs.ruby-lang.org/en/master/contributing/contributing_md.html. You changed the directory structure in #13012.

Reply:
Fixed (I thiink!).